src/main/webapp/css/user-page.css

```diff
@@ -64,3 +68,69 @@ th, td {
   text-align: left;
   border-bottom: 1px solid #ddd;
 }
+
+body {
+  margin: auto;
+  width: 90%;
+  padding: 10px;
+  background: linear-gradient(white, #FF7F50) fixed;
+  font-family: "Avant Garde", Avantgarde, "Century Gothic", CenturyGothic, AppleGothic, sans-serif;
+}
+
+nav {
+  background-color: #FF7F50;
+  font-size: 12pt;
+  padding: 8px;
+}
+
+nav a {
+  color: white;
+  margin-right: 10px;
+}
```

Comment:
this part is repeated in the two files so maybe put it in one css file and include it in the html?

Reply:
Yes, great point thank you! Fixed it :)